kiwi/defaults.py

```diff
@@ -560,7 +573,8 @@ def get_unsigned_grub_loader(root_path):
         """
         unsigned_grub_file_patterns = [
             '/usr/share/grub*/*-efi/grub.efi',
-            '/usr/lib/grub*/*-efi/grub.efi'
+            '/usr/lib/grub*/*-efi/grub.efi',
+            '/boot/efi/EFI/fedora/grubx64.efi'
```

Comment:
This should be `/boot/efi/EFI/*/grubx64.efi`. It's not just Fedora that ships that way. RHEL/CentOS and Mageia do too.

Reply:
yep, done now